readme.md

```diff
@@ -1,2 +1,67 @@
 React Material Icons
 ====================
+
+This package includes all [Material Icons](https://material.io/resources/icons/?style=baseline) as reusable react components
+
+## Table of content
+- [Usage](#usage)
+- [Build & Publish](#build--publish)
+- [Information](#information)
+
+## Usage
+
+In order to use this package you have to install it through npm
+```bash
+npm install @aboutbits/react-material-icons
+```
+
+After you successfully installed the package you can simply use the icons in you react application like this
+```jsx
+import React from 'react'
+
+const MyCommponent = () => {
+    return (
+        <IconCached />
+    );   
+}
+``` 
+
+SVG related parameters like height and width can be passed as props
+
+## Build & Publish
+
+To build the package, simply run the following command
```

Comment:
I would extract the description of the build/clean command into it's own section. And keep the "Build & Publish" section equals to the other packages.

Reply:
Build and clean have got an own section.